gdrive_sync/management/commands/sync_gdrive_files.py

```diff
@@ -55,6 +55,12 @@ def handle(self, *args, **options):
                         continue
 
                     try:
+                        log.info(
```

Comment:
Use `self.stdout.write()` instead of `log.info()`

Reply:
Done